src/commands/add-liquidity.ts

```diff
@@ -0,0 +1,21 @@
+import { BN, Wallet } from '@project-serum/anchor'
+import { Marinade } from '../marinade'
+import { MarinadeConfig } from '../modules/marinade-config'
+import { solToLamportsBN } from '../util/conversion'
+
+export const addLiquidity = async (amount: number, { sol }: { sol: boolean }): Promise<void> => {
```

Comment:
Please rename to:
`export async function addLiquidityAction(amountSol: number): Promise<void> => {`
notes: 
* `export async function` is easier to read
* naming it as `addLiquidityAction` separates it as a function to be called by a Command from commander
* remove `--sol` option, commented code in marinade-cli.ts about `Option("--sol...` was misleading, sorry for that. Let's keep it the same as the Rust counterpart, just a f64 sol amount as parameter

Reply:
How about `Marinade.addLiquidity`/`Marinade.removeLiquidity`? Do we want to keep it consistent with CLI and pass SOLs as well, or are we going with lamports so we can pass integer/BN?